src/qt/walletframe.cpp

```diff
@@ -62,6 +61,14 @@ bool WalletFrame::addWallet(WalletModel *walletModel)
     mapWalletViews[walletModel] = walletView;
 
     connect(walletView, &WalletView::outOfSyncWarningClicked, this, &WalletFrame::outOfSyncWarningClicked);
+    connect(walletView, &WalletView::transactionClicked, gui, &BitcoinGUI::gotoHistoryPage);
+    connect(walletView, &WalletView::coinsSent, gui, &BitcoinGUI::gotoHistoryPage);
+    connect(walletView, &WalletView::message, [this](const QString& title, const QString& message, unsigned int style) {
+        gui->message(title, message, style);
+    });
```

Comment:
s/`[this]`/`[gui]`/ ?

Reply:
Doesn't work, `gui` is a member.

Reply:
oops

`BitcoinGUI::message` is a slot, and there is no need to capture local variable `gui`.
Why not use non-lambda connection syntax?

Reply:
Right, originally couldn't connect to slot because of that. I think we can "clean" these later.